ads/google/imaVideo.js

```diff
@@ -1278,7 +1306,11 @@ export function resetControlsAfterAd() {
  * @visibleForTesting
  */
 export function showControls() {
-  setStyle(controlsDiv, 'display', 'flex');
+  if (!controlsVisible) {
+    setStyle(controlsDiv, 'display', 'flex');
+    controlsVisible = true;
+  }
+
   // Hide controls after 3 seconds
   if (playerState == PlayerStates.PLAYING) {
```

Comment:
I was playing with the `mousemove` event, and it seems that it fires a LOT. I am not sure about the cost of `setTimeout` and `clearTimeout`. Do you think it's fine w/o a throttle?

Reply:
Yeah the `mousemove` event does fire a lot, but I was thinking about it and I essentially came to two conclusions:

1. The event will only be fired on desktop devices, during certain sections of watching the video.
2. Adding a throttle *may* improve user experience, however, I couldn't think of a good way to implement this that would be flexible and testable, thus hurting developer experience.

Though, I do think this is worth having a quick chat about on how we can easily add/remove this dynamic event, that has a throttle. 😄 I'll reach out tommorow, or vice/versa 👍